utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
is sv_easing_functions.py used anywhere else other than the easing.py node? :)
Anyway, I’ll change it. Also should this other dictionary be placed in the sv_easing or in mixNumbers ? I’m thinking that if other nodes will be develop to do the same kind of interpolation like mixNumber nodes, a lot of code would be replicated.